src/docs/src/whatsnew/3.3.rst

```diff
@@ -20,6 +20,56 @@
     :depth: 1
     :local:
 
+.. _release/3.3.3:
+
+Version 3.3.3
+=============
+
+Features and Enhancements
+-------------------------
+
+* :ghissue:`4878`: Add an option to scrub some sensitive headers from external
+  json requests.
+
+* :ghissue:`4847`: Require auth for _replicate endpoint.
```

Comment:
The name of the `_replicate` endpoint could be put in monospaced for better readability.